VotingApplication/VotingApplication.Web/Api/Controllers/ManagePollTypeController.cs

```diff
@@ -39,7 +39,9 @@ public void Put(Guid manageId, ManagePollTypeRequest updateRequest)
                     ThrowError(HttpStatusCode.BadRequest, ModelState);
                 }
 
-                if (updateRequest.PollType.ToLower() != poll.PollType.ToString().ToLower())
+                if (updateRequest.PollType.ToLower() != poll.PollType.ToString().ToLower() ||
+                   (poll.PollType == PollType.Points && (updateRequest.MaxPerVote != poll.MaxPerVote ||
```

Comment:
Looks good, but can we add a test to assert that changing the points value on a non-points poll does not clear the votes out